Features/Swap/Sources/ViewModels/SwapSceneViewModel.swift

```diff
@@ -62,7 +62,8 @@ public final class SwapSceneViewModel {
         wallet: Wallet,
         asset: Asset,
         walletsService: WalletsService,
-        swapQuotesProvider: SwapQuotesProvidable
+        swapQuotesProvider: SwapQuotesProvidable,
+        onSwap: ((TransferData) -> Void)? = nil
```

Comment:
```suggestion
        onSwap: TransferDataAction = nil
```

there is something like this